File: swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/MailBlockRspamd.scala
package io.swagger.server.model


/**
 * = MailBlockRspamd =
 *
 * A block pattern sourced from the rspamd database.  Represents a sender whose recent messages contained suspicious subject lines (e.g. containing relay/proxy strings) repeated more than 4 times in the last 3 days. The `from` address can be passed to `POST /mail/blocks/delete` to delist it.
 *
 * @param from The sender email address.  Pass this value as `email` to `POST /mail/blocks/delete` to delist the sender. for example: ''user@domain.com''
 * @param subject The suspicious subject pattern that triggered the block. for example: ''Test email''
 */
case class MailBlockRspamd (
  from: String,
  subject: String
)

File: swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/mailbaby-client-csharp-dotnet2/Model/MailBlockRspamd.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// A block pattern sourced from the rspamd database.  Represents a sender whose recent messages contained suspicious subject lines (e.g. containing relay/proxy strings) repeated more than 4 times in the last 3 days. The &#x60;from&#x60; address can be passed to &#x60;POST /mail/blocks/delete&#x60; to delist it.
  /// </summary>
  [DataContract]
  public class MailBlockRspamd {
    /// <summary>
    /// The sender email address.  Pass this value as `email` to `POST /mail/blocks/delete` to delist the sender.
    /// </summary>
    /// <value>The sender email address.  Pass this value as `email` to `POST /mail/blocks/delete` to delist the sender.</value>
    [DataMember(Name="from", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "from")]
    public string From { get; set; }

    /// <summary>
    /// The suspicious subject pattern that triggered the block.
    /// </summary>
    /// <value>The suspicious subject pattern that triggered the block.</value>
    [DataMember(Name="subject", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "subject")]
    public string Subject { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MailBlockRspamd {\n");
      sb.Append("  From: ").Append(From).Append("\n");
      sb.Append("  Subject: ").Append(Subject).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
